spec/models/show_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Show, type: :model do
+  describe 'show model' do
+
+    it 'creates a valid show' do
+      show = create(:show)
+      expect(show).to be_valid
+    end
+
+    it 'has a title' do
+      show = described_class.new(title: nil)
+      expect(show).not_to be_valid
```

Comment:
you can even expect show error message to be show, like:
```
show = described_class.new(title: nil)
show.valid?
expect(show.errors[:title]).to include('must be present') #(forgot exact error message)
```

Reply:
or even better, for this default validations, just use https://github.com/thoughtbot/shoulda-matchers and just do:

```
it { should validate_presence_of(:name) }
```

Reply:
Thank you, Goran! Really nice gem 💎